static/js/admin/src/ExportMenu.js

```diff
@@ -83,6 +84,14 @@ define('ExportMenu', [ 'jquery', 'jquery-ui', 'underscore', 'qtip', 'cycle'],
                                     });
                                     sessionStorage.removeItem('exportTargets');
                                     break;
+                                case "exportCSV" :
+                                    self.options.actionHandler.addEvent({
+                                        action : 'ChangeLocation',
+                                        url : `api/exportTree/csv?ids=${self.getMemberOrderTargetIds()}`,
```

Comment:
I don't think `getMemberOrderTargetIds` is what we want to use here, since it only returns work ids. For this function, the user should be able to select any object types except for the root of the repository (which would get rejected in the controller anyways). You could probably just use `getTargets`